MapboxMaps.podspec

```diff
@@ -1,9 +1,9 @@
 Pod::Spec.new do |m|
 
-  version = '10.0.0-beta.12'
+  maps_version = '10.0.0-beta.12'
```

Comment:
With this change, the script in the above mentioned PR might need changing.

Reply:
This is absolutely necessary because otherwise my `sed` step for podspec is going to update swift_version